app/forms/contact_email_form.rb

```diff
@@ -6,6 +6,7 @@ class ContactEmailForm < ApplicationForm
   validates :email, presence: true, on: :deposit
   validates :email, format: {
     with: URI::MailTo::EMAIL_REGEXP,
-    allow_blank: true
+    allow_blank: true,
+    message: 'must provide a valid email address' # rubocop:disable Rails/I18nLocaleTexts
```

Comment:
Does something like: https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/config/locales/en.yml#L42 not work here? It seems if we're disabling a rubocop rule and have a possible approach, I would lean towards agreeing with rubocop. But that's a minor, non-blocking issue.

Reply:
Concur with @aaron-collier

Reply:
I'm going to adjust this and other validation messages in a follow-up PR.